### PR TITLE
+imagemagick

### PR DESCRIPTION
**Pull request by jhheider**

### CODE REVIEW

File: projects/imagemagick.org/package.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+distributable:
+  #FIXME: they use a weird versioning scheme
+  url: https://github.com/ImageMagick/ImageMagick/archive/refs/tags/7.1.0-61.tar.gz
+  strip-components: 1
+
+versions:
+  #FIXME: they use a weird versioning scheme
+  # github: ImageMagick/ImageMagick
+  - 7.1.0.61
```
Comment (jhheider) [inline projects/imagemagick.org/package.yml:9]: @mxcl you ok with this for now? They use the pre-release identifier as a super-patch, or something.

Comment (mxcl) [reply projects/imagemagick.org/package.yml:9]: Fine for now, we will need to figure it out one day tho.

File: projects/imagemagick.org/package.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      --prefix={{ prefix }} \
+      --disable-silent-rules
+    make --jobs {{ hw.concurrency }} install
+
+test:
+  dependencies:
+    curl.se: '*'
+  script: |
+    curl "https://upload.wikimedia.org/wikipedia/commons/6/6a/PNG_Test.png" -o a.png
+    identify a.png | grep "a.png PNG 3120x3920 3120x3920+0+0 8-bit sRGB 7.02443MiB 0.000u"
```
Comment (mxcl) [inline projects/imagemagick.org/package.yml:53]: newlines at the end of files is the convention. Let’s be consistent and do that. Not the end of the world if we forget, but it's a worthwhile goal.

File: projects/imagemagick.org/package.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    freedesktop.org/pkg-config: '*'
+  script: |-
+    # seemingly `./configure` doesn't use pkg-config for libjpeg
+    export CPPFLAGS=`pkg-config --cflags libjpeg`
+    export LDFLAGS=`pkg-config --libs libjpeg`
```
Comment (mxcl) [inline projects/imagemagick.org/package.yml:41]: Could be `env`

**Review by mxcl — APPROVED** [review]

File: projects/imagemagick.org/package.yml (hunk 3 above)
Comment (jhheider) [reply projects/imagemagick.org/package.yml:41]: I was thinking about that. Back ticks should interpret at the right time, right?

**Pull request merged**